components/identity-mgt/org.wso2.carbon.identity.mgt.endpoint.util/src/main/java/org/wso2/carbon/identity/mgt/endpoint/util/client/api/RecoveryApiV2.java

```diff
@@ -193,4 +247,76 @@ private RecoveryResponse recover(RecoveryRequest recoveryRequest, String tenantD
                 localVarHeaderParams, localVarFormParams, localVarAccept, localVarContentType, localVarAuthNames,
                 localVarReturnType);
     }
+
+    private ResendResponse resend(ResendRequest resendRequest, String tenantDomain, Map<String, String> headers,
+                                  String localVarPath) throws ApiException {
+
+        // Verify if the required parameter 'recoveryRequest' is set.
+        if (resendRequest == null) {
+            throw new ApiException(400, "Missing the required parameter 'resendRequest' when requesting recovery");
+        }
+        if (StringUtils.isBlank(tenantDomain)) {
+            tenantDomain = MultitenantConstants.SUPER_TENANT_DOMAIN_NAME;
+        }
+        basePath = IdentityManagementEndpointUtil.getBasePath(tenantDomain,
+                IdentityManagementEndpointConstants.UserInfoRecovery.RECOVERY_API_V2_RELATIVE_PATH);
+        apiClient.setBasePath(basePath);
+        Map<String, String> localVarHeaderParams = new HashMap<>();
+        if (MapUtils.isNotEmpty(headers)) {
+            localVarHeaderParams.putAll(headers);
+        }
+        final String localVarAccept = apiClient.selectHeaderAccept(new String[]{"application/json"});
```

Comment:
We could define a constant for `application/json`

Reply:
addressed with https://github.com/wso2/carbon-identity-framework/pull/5601/commits/b1c3d31e9ef460ab19ff45869ebe4e8dec8454d8